src/style.css

```
@@ -1,3 +1,87 @@
+html {
+  height: 100%;
+  width: 100%;
+}
+
 body {
   margin: 0;
 }
+
+@media (max-width: 1200px) {
+  body {
+    width: 1200px;
+    font-size: 12px;
+    font-weight: 500;
+  }
+}
+
+@media (max-width: 1024px) {
+  body {
+    width: 1024px;
+  }
+}
+
+.header {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  height: 60px;
+  margin: 10px;
+  align-items: center;
+  justify-content: space-between;
+  background-color: rgba(255, 255, 255, 1);
+  position: relative;
+}
+
+.images {
+  width: 40px;
+  height: 40px;
+  margin: 0 50px;
+  flex-shrink: 0;
+}
+
+.nav-item {
+  padding: 0;
+  line-height: 60px;
+  color: #000;
+  margin: 0 10px;
+  font-weight: 500;
+}
```

```suggestion
.nav-item:not(:last-child) {
  margin-right: 20px;
}
```
nav-item should be responsible only for spaces to each other. spaces should be only between .nav-item